spacy_llm/tests/tasks/examples/ner_inconsistent.yml

```
@@ -0,0 +1,11 @@
+- text: Jack and Jill went up the hill.
+  entities:
+    PERSON:
+      - Jack
+      - Jill
+    location:
```

```suggestion
    LOCATION:
```
(it doesn't really matter, but just for consistency)